packages/nx-plugin/src/executors/cli/executor.ts

```diff
@@ -11,7 +14,8 @@ export type ExecutorOutput = {
   error?: Error;
 };
 
-export default function runAutorunExecutor(
+// eslint-disable-next-line max-lines-per-function
```

Comment:
See previous comment.

```suggestion
```

Reply:
Really nice you also added this one that is connected! <3